include/jemalloc/internal/hpa_opts.h

```diff
@@ -59,6 +104,43 @@ struct hpa_shard_opts_s {
 	 * Maximum number of hugepages to purge on each purging attempt.
 	 */
 	ssize_t experimental_max_purge_nhp;
+
+	/*
+	 * When the number of inactive bytes in a hugepage is >=
+	 * purge_threshold, the page is purgable.  Setting this to HUGEPAGE
+	 * would only purge empty pages if using hugify_style_eager or
+	 * hugify_style_eager_and_lazy is true and the purges would be exactly
+	 * HUGEPAGE bytes.  Depending on your kernel settings, this may result
+	 * in better performance.  When purge_threshold is < HUGEPAGE and
+	 * hugify style is hugify_style_lazy, empty pages will be purgable even
+	 * when below threshold.
+	 *
+	 * Please note, when threshold is reached, we will purge all the dirty
+	 * bytes, and not just up to the threshold.  If this value is 1, then
+	 * all the pages are purgable.  We treat purgability constraint for
```

Comment:
A magical value of `1` seems a little confusing since this value is denominated in bytes and `1` is a valid number of bytes.